demo/demo.sh

```diff
@@ -4,8 +4,6 @@ THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 ROOT="${THIS_DIR}/../"
 SERVER_DIR="${ROOT}/pipeline_server/"
 
-service postgresql restart
-
 # Kill previous server instance if any.
 pkill -9 dbsp_pipeline_server
```

Comment:
Do you remember why this was needed.  If we indeed must restart postgres, we should `pkill` the server first, since it might have a live connection to postgres (no need to make any changes, I am in the process of cherry-picking this PR in a separate branch, so just trying to understand)